### PR TITLE
docs: add favicon

### PR DESCRIPTION
**Pull request by jfatta**
Add auth0 favicon to docs homepage

image

### CODE REVIEW

File: docs/_includes/disqus_comments.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{%- if page.comments != false and jekyll.environment == "production" -%}
```
Comment (Widcket) [inline docs/_includes/disqus_comments.html:1]: Is this file necessary if we don't use Disqus?

File: docs/_includes/footer.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<footer class="site-footer h-card">
```
Comment (Widcket) [inline docs/_includes/footer.html:1]: Why are we adding a custom footer?

File: docs/_includes/google-analytics.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<script>
```
Comment (Widcket) [inline docs/_includes/google-analytics.html:1]: Should we add this?

File: docs/_includes/head.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {%- seo -%}
+  <link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">
+  <link rel="icon" type="image/png" href="favicon.png">
+  {%- feed_meta -%}
+  {%- if jekyll.environment == 'production' and site.google_analytics -%}
+    {%- include google-analytics.html -%}
```
Comment (Widcket) [inline docs/_includes/head.html:10]: Same as above, do we need GA?

File: docs/_includes/social.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<ul class="social-media-list">
```
Comment (Widcket) [inline docs/_includes/social.html:1]: What are these for? Do we want this?

Comment (Widcket) [reply docs/_includes/social.html:1]: Same for the above individual social icons.

Comment (jfatta) [reply docs/_includes/social.html:1]: @Widcket, yes. To override **one** include, we need to respect the entire folder structure here https://github.com/jekyll/minima/tree/master/_includes. It is a common practice with github themes until you have your own. I tried adding only the header to our _includes folder and the `make start-doc` process fails.

**Review by Widcket — APPROVED** [review]
